agent/util-scripts/pbench-metadata-log

```diff
@@ -118,8 +122,10 @@ EOF
         cat >> $log <<EOF
 [tools/$remote]
 EOF
-        for tool in $(ssh $ssh_opts $remote /bin/ls $pbench_run/tools-$group) ;do
-            echo $tool: $(ssh $ssh_opts $remote cat $pbench_run/tools-$group/$tool | sed 's/^/ /')
+        mkdir -p $tmpdir/$remote
+        scp $ssh_options -r $remote:$pbench_run/tools-$group $tmpdir/$remote > /dev/null
```

Comment:
In the unit tests we mock out scp so the following `for` loop doesn't do anything, do you think we could enhance that mock to provide a `tools-$group` directory which would enable us to see the output generated?